pkg/scaffold/controller/controllersuitetest.go

```diff
@@ -47,7 +47,7 @@ var controllerSuiteTestTemplate = `{{ .Boilerplate }}
 package {{ lower .Resource.Kind }}
 
 import (
-	"log"
+	stdlog "log"
```

Comment:
For reviewer:
package name "log" is conflicted with the variable "log" in controller.go, so I choose `stdlog` to instead of it.

Reply:
What is the reason of using golang log pkg instead of controller-runtime's log pkg here?

Reply:
I think log pkg is good enough for **tests**. 
If should use controller-runtime's log here, I can change it.
Thanks.